addons/website/views/snippets/snippets.xml

```diff
@@ -136,8 +136,14 @@
                   data-select-style="0" data-css-property="border-width"
                   data-unit="px" data-extra-class="border"/>
     </we-row>
+    <we-select data-css-property="border-style" data-dependencies="border_width_opt">
+        <we-button data-select-style="solid">Solid</we-button>
+        <we-button data-select-style="dashed">Dashed</we-button>
+        <we-button data-select-style="dotted">Dotted</we-button>
+        <we-button data-select-style="double">Double</we-button>
+    </we-select>
```

Comment:
Can't this be on the same as width and color? The left panel size will get bigger.

Reply:
I put it between the border color and width initially, I think I moved it while trying to figure out why it wasn't working (it was the css shorthand thing), but yeah it's much better on one line.